VLogProcessor/src/edu/umich/robustopt/dblogin/DatabaseLoginConfiguration.java

```diff
@@ -269,7 +269,7 @@ public Object clone() throws CloneNotSupportedException {
     // each DB vendor implements the following function accordingly!
 	public abstract Connection createConnection() throws Exception;
 	
-	public abstract Map<String, Schema> getSchemaMap() throws SQLException; 
+	public abstract Map<String, Schema> getSchemaMap() throws Exception;
```

Comment:
Because of [here](https://github.com/mozafari/cliffguard/pull/1/files#diff-84e647f5ba178092c89b50766a6376d9R44), we need to throw `Exception` instead of `SQLException`